### PR TITLE
KFSPTS-19994: Remove customization allowing Fiscal Officers to adjust accounts that are not theirs on REQS.

### PR DESCRIPTION
**Pull request by nkimble**

### CODE REVIEW

**Comment by nkimble** [comment]
This PR is ready for review by any developer with available cycles.
The numerous formatting changes in class "CuRequisitionServiceImpl.java" is due to manually rebasing those specific methods to their respective base code versions to ensure proper processing.

File: src/main/java/edu/cornell/kfs/module/purap/document/service/impl/CuRequisitionServiceImpl.java [hunk 1]
```diff
@@ -82,115 +85,144 @@ protected String checkAutomaticPurchaseOrderRules(RequisitionDocument requisitio
...
+                if (item.getItemUnitPrice() != null && BigDecimal.ZERO.compareTo(item.getItemUnitPrice()) != 0) {
                     // discount or trade-in item has unit price that is not empty or zero
-                    return "Requisition contains a " + item.getItemType().getItemTypeDescription() + " item, so it does not qualify as an APO.";
+                    return "Requisition contains a " + item.getItemType().getItemTypeDescription() +
+                            " item, so it does not qualify as an APO.";
                 }
             }
-            
-            
+//            //Base code logic check not in CU mod
```
Comment (jhulslander) [inline src/main/java/edu/cornell/kfs/module/purap/document/service/impl/CuRequisitionServiceImpl.java:158]: should this commented out code remain?

**Review by jhulslander — COMMENTED** [review]
code changes look good, I just had a question about some commented out code

File: src/main/java/edu/cornell/kfs/module/purap/document/service/impl/CuRequisitionServiceImpl.java (hunk 1 above)
Comment (nkimble) [reply src/main/java/edu/cornell/kfs/module/purap/document/service/impl/CuRequisitionServiceImpl.java:158]: The purpose of these changes are to start getting us back to base code.
When the next mod in the purap area needs to be removed, IMHO is will be easier to pinpoint base code versus CU code in this method if these comments are left.

Comment (jhulslander) [reply src/main/java/edu/cornell/kfs/module/purap/document/service/impl/CuRequisitionServiceImpl.java:158]: hmmm, I guess we can try and see how it goes

**Review by jhulslander — APPROVED** [review]

**Pull request merged**